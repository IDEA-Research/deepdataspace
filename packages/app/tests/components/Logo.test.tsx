import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Logo from '@/components/Logo';

test('render GlobalLoading by snapshot', () => {
  const { container } = render(
    <MemoryRouter>
      <Logo />
    </MemoryRouter>,
  );
  expect(container).toMatchSnapshot();
});
