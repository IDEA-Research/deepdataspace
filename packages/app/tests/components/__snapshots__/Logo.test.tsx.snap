// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render GlobalLoading by snapshot 1`] = `
<div>
  <a
    href="/"
  >
    <img
      alt="logo"
      class="logo"
      src="logo_title.png"
    />
  </a>
</div>
`;
